src/components/include/test/application_manager/mock_rpc_service.h

```diff
@@ -43,6 +43,8 @@ class MockRPCService : public application_manager::rpc_service::RPCService {
                void(const hmi_apis::FunctionID::eType& function_id,
                     const hmi_apis::messageType::eType& message_type,
                     const std::map<std::string, SMember>& members));
+
+  MOCK_METHOD0(Stop, void());
```

Comment:
@mked-luxoft I would add 2 unit tests : 
1. In application manager to check that rpc_service_->Stop() on ApplicationManagerImpl::Stop
2. In RPC Service to check that message queue was stopped

Reply:
1. added in 8bbe3f46780d4091192aa5bdacf05acddab4ca82
2. as far as i understand, there is no UT file for `RPCServiceImpl`